packages/widgets/src/components/CollapsibleNavBar/types.ts

```diff
@@ -0,0 +1,20 @@
+import { IMapIcons } from '@ssa-ui-kit/core';
+
+export type CollapsibleNavBarItem = {
```

Comment:
What do you think about extending these types from `NavBarExtendedItem`, and `NavBarExtendedGroup` respectively?